pages/foundation/index.html

```diff
@@ -39,6 +39,11 @@ <h3>Community Member Sponsorship</h3>
           <p><a href="sponsorship.html#community">Learn more about Community Member Sponsorship</a></p>
             <p>See all the current supporters on our <a href="/foundation/sponsor-list.html">Sponsors List</a>.</p>
 
+            <h3>Tools and Infrastructure Partners</h3>
+            <p>The Grails Foundation is truly appreciative of the tremendous support it receives
+                from these organizations. Their generous contributions of development tools, infrastructure,
```

Comment:
Can we add a link to "these organizations" that points to the section of the sponsors page that displays these partners like we did on the Micronaut site?